src/stores/uiOrderStore.ts

```diff
@@ -212,6 +227,64 @@ class UiOrderStore extends BaseStore {
           precisionFloor(+convertedBalance, this.quantityAccuracy);
   };
 
+  setMarketTotal = (
+    operationVolume?: string | number,
+    operationType?: Side,
+    debounce?: boolean
+  ) => {
+    if (
+      (!operationVolume && !operationType && !this.marketTotal.canBeUpdated) ||
+      (operationVolume &&
+        operationType &&
+        !this.marketTotal.canBeUpdated &&
+        debounce)
+    ) {
+      return;
+    } else if ((!operationVolume && !operationType) || debounce) {
+      this.marketTotal.canBeUpdated = false;
+      setTimeout(
+        () => (this.marketTotal.canBeUpdated = true),
+        MARKET_TOTAL_DEBOUNCE
+      );
+    }
+
+    if (operationVolume || operationVolume === 0) {
+      this.marketTotal.operationVolume =
+        typeof operationVolume === 'number'
```

Comment:
I think typeof is redundant
```this.marketTotal.operationVolume = parseFloat(operationVolume)``` should work the same

Reply:
I've tried to do that but got typescript error for parseFloat: "Type 'number' is not assignable to type 'string'"